packages/deployment/iam/cloudformation.template.yml

```diff
@@ -317,6 +317,180 @@ Resources:
                   - cloudformation:DescribeStacks
                 Resource: '*'
 
+  # Role with additional permissions for migrations
+  MigrationProcessingRole:
```

Comment:
So I feel like this _could_ go into the lambda processing role. Which additional perms are needed? I guess it is probably better to keep it separate and this will help moving forward.

Reply:
@laurenfrederick  it could, it's literally only adding lambda:CreateEventSourceMapping and lambda:ListEventSourceMappings.....   we could just add it to the lambda processing role, but the separation of concerns feels better going forward.   If anything, we might be able to *reduce* perms on this one, but that felt like it was better relegated to a migration improvement ticket.